src/introduction.html

```diff
@@ -260,33 +260,46 @@ <h4 id="intro_goals">Design Goals of MathML</h4>
       <h3 id="intro_overview">Overview</h3>
       
       <p>
-         <span>MathML is a markup language for describing mathematics.
-            It is usually expressed in XML syntax, although HTML and other syntaxes are possible.</span>
-         A special aspect of MathML is that there are two main strains of markup:
-         Presentation markup, discussed in <a href="#presm"></a>,
-         is used to display mathematical expressions;
-         and Content markup, discussed in <a href="#contm"></a>,
-         is used to convey mathematical meaning.
-         Content markup is specified in particular detail.   
-         This specification makes use of
-         <span>an XML</span>
-         format called Content Dictionaries
-         This format has been developed by the
-         OpenMath Society, [[OpenMath2004]] with the dictionaries being used by this specification involving joint development
-         by the OpenMath Society and the W3C Math Working Group.
+         <span>MathML is a markup language for describing mathematics.</span>
+         It uses XML syntax when used standalone or within other XML,
+         or HTML syntax when used within HTML documents.
+         Conceptually, MathML consists of two main strains of markup:
+         Presentation markup is used to display mathematical expressions;
+         and Content markup is used to convey mathematical meaning.
+         These two strains, along with other external representions,
+         can be combined using parallel markup.
          </p>
       
-      
-      <p>Fundamentals common to both strains of markup
-         are covered in <a href="#fund"></a>,
-         while the means for combining these strains,
-         as well as external markup, into single MathML objects
-         are discussed in <a href="#mixing"></a>.
+      <p>
+         The specification of MathML is developed in two layers.
+         <span>MathML Core</span> covers (most of) Presentation Markup,
+         with the focus being the display of mathematics in web browsers.
+         <span>MathML Full</span>, this specification, extends <span>MathML Core</span>
+         primarily by defining Content MathML, in <a href="#contm"></a>.
+         It also defines extensions to Presentation MathML, discussed in <a href="#presm"></a>.
+         These extensions consist of additional attributes, elements or
+         enhanced syntax of attributes.
+         These are defined for compatibility with legacy MathML,
+         as well as to cover more complex notations.
```

Comment:
I wonder if `less common` or `harder to implement` is a better emphasis than `more complex` here, as one can compose together MathML Core to cover some really complex notations already.

I worry that the sentence may sound discouraging that e.g. MathML Core shouldn't be used for higher mathematics.

Reply:
sounds good; done!